common-tasks/backup-emergency-restore-v4.md

```diff
@@ -39,6 +39,8 @@ The Qubes backup system has been designed with emergency disaster recovery in mi
         vm1/whitelisted-appmenus.list.000.enc
         dom0-home/dom0user.000.enc
 
+    **Note:** For backups with multiple VM, find the VM <-> id correspondence in `qubes.xml.000.enc`, extract required files `tar -i -xvf qubes-backup-2015-06-05T123456 backup-header backup-header.hmac vm1/`.
```

Comment:
Sorry, I don't really understand this sentence. Could you try rewriting it to make it more grammatical?

Reply:
No problem, tried to keep it as short as possible but it needed some rephrasing. Changed it.